os/tools/mkbootparam.py

```diff
@@ -103,12 +109,22 @@ def make_bootparam():
 
     with open(bootparam_file_path, 'wb') as fp:
         # Write Data
+        bootparam_version = 0
+        bootparam_format_version = 1.0
```

Comment:
Is it okay to use only 4bytes for float type?
As I know, default float type size is 8bytes in python.

Reply:
Yes, a size of float is 4bytes in python. Double size is 8bytes